### PR TITLE
Update `exampleapp` to use new routing/templates settings

### PR DESCRIPTION
**Pull request by wm3ndez**
The exampleapp is still running deprecated code.

### CODE REVIEW

File: exampleapp/urls.py [hunk 1]
```diff
@@ -1,16 +1,15 @@
-from django.conf.urls import patterns, include
+from django.urls import path, include
```
Comment (hramezani) [inline exampleapp/urls.py:1]:
```suggestion
from django.urls import include, re_path
```

File: exampleapp/urls.py [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 admin.autodiscover()
 
-urlpatterns = patterns(
-    "",
-    (r"^admin/", include(admin.site.urls)),
-    (r"^admin/defender/", include("defender.urls")),
-)
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("admin/defender/", include("defender.urls")),
```
Comment (hramezani) [inline exampleapp/urls.py:11]:
```suggestion
    re_path(r"^admin/", admin.site.urls),
    re_path(r"^admin/defender/", include("defender.urls")),
```

Comment (wm3ndez) [reply exampleapp/urls.py]: Changes applied

**Review by hramezani — APPROVED** [review]
LGTM!

**Pull request merged**